File: official/docs/java/v5/rates/regenerate.java
package shipments;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Shipment;

public class Regenerate {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = "EASYPOST_API_KEY";

        Shipment shipment = Shipment.retrieve("shp_...");

        shipment = shipment.newRates();

        System.out.println(shipment);
    }
}

File: official/docs/golang/v4/rates/regenerate.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func regenerate() {
	client := easypost.New("EASYPOST_API_KEY")

	shipment, _ := client.RerateShipment("shp_...")

	fmt.Println(shipment)
}
